src/midi/MidiMessage.cpp

```diff
@@ -16,6 +16,10 @@
 #include <vector>
 #include <iostream>
 
+#ifdef _WIN32
+#include <iterator>
```

Comment:
This needs do to be modified in https://github.com/craigsapp/midifile . Maybe @craigsapp will make it without a pull request?